apps/webapp/app/services/externalApis/integrations/linear.ts

```diff
@@ -0,0 +1,335 @@
+import type { HelpSample, Integration, ScopeAnnotation } from "../types";
+
+const repoAnnotation: ScopeAnnotation = {
```

Comment:
this file looks to be just a duplicate of the Github one

Reply:
Yes, just needed a quick placeholder for testing, already on my todos